main.tf

```diff
@@ -35,7 +35,7 @@ resource "azurerm_app_service_plan" "main" {
 }
 
 resource "azurerm_app_service" "main" {
- name = "module12-azure-terraform-sj"
+ name = "module13-azure-terraform-sj"
```

Comment:
(There's no need to update this every module, but equally no harm now that we have a nice automated way to do so!)

Reply:
Thanks